kolibri/plugins/coach/assets/src/views/reports/ReportsGroupReportQuizLearnerListPage.vue

```diff
@@ -27,16 +27,15 @@
           <transition-group tag="tbody" name="list">
             <tr v-for="tableRow in table" :key="tableRow.id">
               <td>
-                <KLabeledIcon icon="person">
-                  <KRouterLink
-                    v-if="tableRow.statusObj.status !== STATUSES.notStarted"
-                    :text="tableRow.name"
-                    :to="detailLink(tableRow.id)"
-                  />
-                  <template v-else>
-                    {{ tableRow.name }}
-                  </template>
-                </KLabeledIcon>
+                <KRouterLink
+                  v-if="tableRow.statusObj.status !== STATUSES.notStarted"
+                  :text="tableRow.name"
+                  :to="detailLink(tableRow.id)"
+                  icon="person"
+                />
+                <template v-else>
+                  {{ tableRow.name }}
+                </template>
```

Comment:
this looks like a regression to me - it will lose the icon in a non-linked state.

I would expect something more like:

```vue
<KRouterLink
  v-if="tableRow.statusObj.status !== STATUSES.notStarted"
  :text="tableRow.name"
  :to="detailLink(tableRow.id)"
  icon="person"
/>
<KLabeledIcon v-else icon="person">
  {{ tableRow.name }}
</KLabeledIcon>
```

Reply:
I just found a couple more of these. I'll make a patch right now to fix them.

Reply:
Thanks all - sorry I missed those!